static/styles/dashboard/dashboard.scss

```diff
@@ -298,3 +298,22 @@
 		max-width: 85%;
 	}
 }
+
+.bettermarks-notification {
+	display: flex;
+	color: #424242;
+	border-style: solid;
```

Comment:
nit: you can merge these two css properties into 1 line: border: solid $secondaryColor;